lib/knapsack_pro/adapters/base_adapter.rb

```diff
@@ -60,13 +60,13 @@ def bind
         File.write(self.class.adapter_bind_method_called_file, nil)
 
         if KnapsackPro::Config::Env.recording_enabled?
-          KnapsackPro.logger.debug('Test suite time execution recording enabled.')
+          KnapsackPro.logger.debug('[Regular Mode] Test suite execution time recording enabled.')
```

Comment:
I simplified it more because there is already [knapsack_pro] prefix produced by a logger.

The message on the output looks like this:

```
DEBUG -- : [knapsack_pro] Queue Mode enabled.
```

done: https://github.com/KnapsackPro/knapsack_pro-ruby/pull/237/commits/64ddfabfca1146488558394dda3736aeb18152a0